lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -40,9 +40,6 @@ def setup_rabbitmq_connection_config
         # Specify TLS version if using TLS e.g "TLSv1.2"
         config :ssl, :validate => :string
 
-        # Validate SSL certificate
-        config :verify_ssl, :validate => :boolean, :default => false
```

Comment:
@jordansissel I had the same thought at first. My worry though is that if someone sets it to true and it does nothing that's actually dangerous, so we should probably just remove it.

A larger problem we should tackle (in a different PR) is perhaps making SSL validation the default. This is actually kind of tricky in RabbitMQ Java, so I think its not something we should do immediately. But we should do it. 

Your thoughts?